securedrop_client/export.py

```diff
@@ -184,8 +184,9 @@ def _create_archive(
         with tarfile.open(archive_path, "w:gz") as archive:
             cls._add_virtual_file_to_archive(archive, cls.METADATA_FN, metadata)
 
+            needs_disambiguation = len(filepaths) > 1
```

Comment:
edit: having read your comment above, the `is_multifile_export` name may be a slightly inaccurate choice, but I'm still in favour of naming for what it is/does (is_name_collision), vs what we think of it (needs_disambiguation), and/or adding a comment explaining how we resolve this once we decide on the correct resolution

Reply:
What do you think of an imperative: `prevent_name_collisions` ? @rocodes 

(We don't _know_ name collisions will happen if multiple files are being exported, we only know they're likely.)

Reply:
You know what I'm gonna say...whatever you decide is cool as long as you document it :)  Stylistically, based on what you said I'd go with an `is_possible_name_collision`, but I really don't feel strongly as long as someone new reading the code understands quickly what's happening, here and in add-file-to-archive

Reply:
I renamed the variable to be close to the condition (`is_one_of_multiple_files`) and added a comment explaining why that's relevant.

I renamed the `disambiguate` option to `prevent_name_collisions` because the method is called `add_file_to_archive` and that seemed like a reasonable name for an option.